src/parsers/csvParser.ts

```diff
@@ -0,0 +1,14 @@
+import { parse } from 'csv-parse/sync';
+import { promises as fs } from 'fs';
+
+export class CsvParser {
```

Comment:
not a big deal, but I'd consider just exporting the parseCSV function directly and forget exporting the class

that way, it can be used as:

```
import { parseCSV } from ./parsers/csvParser;

parseCSV(....);
```